src/schemas/catchment_area.py

```diff
@@ -278,6 +279,16 @@ class ICatchmentAreaActiveMobility(BaseModel):
         title="Scenario ID",
         description="The ID of the scenario that is to be applied on the input layer or base network.",
     )
+    street_network_edge_layer_project_id: int = Field(
+        ...,
+        title="Street Network Edge Layer Project ID",
+        description="The layer project ID of the street network edge layer.",
+    )
+    street_network_node_layer_project_id: int = Field(
+        settings.STREET_NETWORK_NODE_DEFAULT_LAYER_PROJECT_ID,
+        title="Street Network Node Layer Project ID",
+        description="The layer project ID of the street network node layer.",
+    )
```

Comment:
I think we can make the _street_network_edge_layer_project_id_ and _street_network_node_layer_project_id_ optional here and hardcode it in the api for now. As the network id can be a bit confusing for api usage without the ui.

Reply:
Sure, will make this change. But it'll need a dynamic _street_network_edge_layer_project_id_ whenever a network scenario is applied.

Reply:
Yes, this I can pass in the client.